### PR TITLE
feat: enforce otp in prod environment

### PR DESCRIPTION
**Pull request by junminahn**

### CODE REVIEW

File: keycloak/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Purpose
+
+This common action covers the one of the limitations of Terraform, such that there is no way to control of existing resources, and ensures the target Keycloak instances to be provisioned as:
```
Comment (thegentlemanphysicist) [inline keycloak/README.md]: Verbiage is strange here, suggested "This common action covers one of the limitations of Terraform, such that there is no way to control existing resources, and it ensures the target Keycloak instances are provisioned as:"
(is this what you meant?)

File: keycloak/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+1. Disable `review profile config` execution of the default authentication flow `first broker login`.
```
Comment (thegentlemanphysicist) [inline keycloak/README.md]: I don't really understand what these enumerated points mean.  Are these the the provisioning steps mentioned in the first sentence, or are these the instances being provisioned?

File: keycloak/README.md (hunk 1 above)
Comment (junminahn) [reply keycloak/README.md]: @thegentlemanphysicist I updated it as you suggested. thanks!

File: keycloak/README.md (hunk 2 above)
Comment (junminahn) [reply keycloak/README.md]: hm. I updated the description. hope it is a little bit clearer.

**Review by thegentlemanphysicist — APPROVED** [review]
thanks for the fix, much more clear now

**Pull request merged**